templates/re-base/finishInstall.js

```diff
@@ -6,6 +6,10 @@ const {execSync} = require('child_process');
 const {existsSync, readFileSync, renameSync, unlinkSync, writeFileSync} = require('fs');
 const {basename, resolve} = require('path');
 
+
+const install = "node scripts/install.js"
```

Comment:
Re-integrate this in the `scripts` field, we don't need it as a constant anymore.

Reply:
Should've caught this!